docs/pages/docs/guides/schema-extension.md

```diff
@@ -1,6 +1,125 @@
 ---
-title: "Schema Extension"
-description: "We've planned this page but not had a chance to write it yet."
+title: "GraphQL Schema Extension"
+description: "Learn how to extend your GraphQL Schema using extendGraphqlSchema."
 ---
 
-{% coming-soon /%}
+Keystone automatically generates a GraphQL Schema based on your [Keystone config](../config/config). This Schema contains all GraphQL Types, Queries and Mutations based on your models and ends up as the generated `schema.graphql` file found in the root of your Keystone project.
```

Comment:
Why are schema, types, queries and mutations capitalised here?

Reply:
Good Question...

Resolved now.